coinbaseadvanced/models/products.py

```diff
@@ -20,6 +20,7 @@ class ProductType(Enum):
 
     SPOT = "SPOT"
 
+Gran = {"ONE_MINUTE": 1, "FIVE_MINUTE": 5, "FIFTEEN_MINUTE": 15, "THIRTY_MINUTE": 30, "ONE_HOUR": 60, "TWO_HOUR": 120, "SIX_HOUR": 360, "ONE_DAY": 720}
```

Comment:
Should we be more explicit with the naming here for clarity? Maybe something like `GRANULARITY_MAP_IN_MINUTES` instead of "Gran"?

Reply:
That make better sense.